pkg/apis/migration/v1alpha1/migmigration_types.go

```diff
@@ -39,6 +39,7 @@ type MigMigrationStatus struct {
 	MigrationCompleted  bool         `json:"migrationCompleted,omitempty"`
 	StartTimestamp      *metav1.Time `json:"startTimestamp,omitempty"`
 	CompletionTimestamp *metav1.Time `json:"completionTimestamp,omitempty"`
+	TaskPhase           string       `json:"taskPhase,omitempty"`
```

Comment:
Should we use `Phase` here? Seems more standard, unless you were purposely differentiating.

Reply:
Perhaps.  Since we were not convinced about exposing the `phase` I thought it best to make it clear that it was different.  That said, I can be easily convinced to name it `Phase`.  @dymurray?

Reply:
I'm a little indifferent... but I think since we were hesitant about using the phase field I would keep it different for now